### DISCUSSION TITLE
How to purchase?

### DISCUSSION THREAD

**Discussion by shawngiese** [GetPublii/Publii#1876] [General]
I decided to try and purchase 12 month access to a theme to support the efforts but I cannot figure it out. The workflow says need to Sign In... but I could not get:
* sign in to complete... though not even sure I have an account already... maybe confusing with login to GitHub for these discussions
* reset of password to work (I only see a sign in button and that is not working)
* make a new account was never an option 

So I cannot complete a purchase. Any ideas?

Also, in trying to purchase, I sometimes click the same item twice... but the shopping cart does not have an option to reduce the quantity of a desired item... so I have to keep restarting the purchase from scratch to get back to x1.

**Comment by bobmitro** [comment]
Simply fill out the form on the checkout page, and the account will be created automatically, which you will be notified about via email. The login option is intended for returning customers, as clearly explained on the checkout page.

**Comment by shawngiese** [comment]
Well, okay then. It was not that clear to me. Maybe the UI could be a little more clear was all I am saying. 
![image](https://github.com/user-attachments/assets/ac2f3895-e742-49a4-93d5-9b2719bded11)
